### ISSUE TITLE
sending email using domcloud

### ISSUE THREAD

**Issue by elycheikhsmail** [domcloud/domcloud#54]
I know it is possible to send email on domcloud using one of  SMTP providers but if possible to get SMTP service included in domcloud features it will be nice, for example it will be possible to host most of CMS based web site using domcloud without need for extrenal service which give a good dx.

**Comment by willnode** [comment]
No, it's complicated. I'm 100% sure if I do it anyway it will directly go into spam.

**Comment by elycheikhsmail** [comment]
is there some specific SMTP provider you advice to use with domcloud ?

**Comment by willnode** [comment]
I use https://www.brevo.com/ and https://forwardemail.net/ both are used in DOM Cloud.
